### PR TITLE
Allow DR user to get a token.

### PR DESCRIPTION
**Pull request by MancunianSam**
This isn't related to any specific repository, it just allows them to
call get-authorization-token.

### CODE REVIEW

File: templates/iam_policy/disaster_recovery_policy.json.tpl [hunk 1]
```diff
@@ -40,6 +40,11 @@
         "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery"
       ],
       "Sid": "writeLogs"
+    },
+    {
+      "Action" : "ecr:GetAuthorizationToken",
+      "Effect": "Allow",
+      "Resource": "*"
```
Comment (techncl) [inline templates/iam_policy/disaster_recovery_policy.json.tpl:47]: Should we not limit the resources?

**Review by techncl — COMMENTED** [review]
Looks good! Just a minor comment.

File: templates/iam_policy/disaster_recovery_policy.json.tpl (hunk 1 above)
Comment (MancunianSam) [reply templates/iam_policy/disaster_recovery_policy.json.tpl:47]: `GetAuthorizationToken` works with "*". You can't restrict it to certain resources.

**Review by techncl — DISMISSED** [review]

**Review by techncl — APPROVED** [review]
👍

**Pull request merged**